### DISCUSSION TITLE
更新日志 - 20230407

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#698] [Announcements]
# 应用：
* QuickStart：
  1. 增加自动配置旁路由的向导
  2. 旁路由向导增加 IPv6 自动配置开关（即是否在 LAN 口启用 DHCPv6 客户端）
  3. 修复自动获取的 DNS 在首页不显示的问题
  4. 下载服务配置增加配置文件夹选项，可将配置文件写入硬盘，防止重启或者刷机丢任务

# iStore 插件：
* 访问AP/光猫：新增。让局域网客户端访问不同子网的AP或光猫。有需要可在 iStore 安装
* MAC地址生成器：新增。方便地修改 WAN 口 MAC 地址，可指定前缀随机生成 MAC 地址。有需要可在 iStore 安装

**Comment by jiyeon-2** [comment]
自动获取的 DNS 设置不了
爱快+ istoreos

**Reply by jjm2473** [reply]
什么意思，截图看看

**Reply by jiyeon-2** [reply]
![Screenshot_2023-04-10-06-50-34-25_df198e732186825c8df26e3c5a10d7cd](https://user-images.githubusercontent.com/50439042/230800074-717a53d4-7314-43b9-bc1b-b89ac6bb5778.jpg)
![Screenshot_2023-04-10-06-50-27-69_df198e732186825c8df26e3c5a10d7cd](https://user-images.githubusercontent.com/50439042/230800076-6cd1973e-965f-4d99-b93d-d984db76a80e.jpg)
我想自动获取DNS

**Reply by jjm2473** [reply]
如果你的wan口是静态ip，那就没办法自动获取dns啊，这不是常识吗，你自己电脑也是这样的。如果要自动获取dns，那要把wan口的ip也改成dhcp自动获取。你现在是旁路由模式，那一般都不会使用dhcp的了，也就是没法自动获取ip和dns。

**Comment by bdpqwm** [comment]


**Reply by jjm2473** [reply]
你划线的日志是关机过程中调用的，你应该往前翻，看看什么时候开始关机的日志。
